src/main/host/thread_ptrace.c

```diff
@@ -299,9 +374,12 @@ static void _threadptrace_enterStateSignalled(ThreadPtrace* thread,
             error("ptrace: %s", g_strerror(errno));
             return;
         }
+        debug("threadptrace_enterStateSignalled regs: rip=%llx rax=%llx arg0=%llx arg1=%llx "
+              "arg2=%llx arg3=%llx arg4=%llx arg5=%llx",
+              regs.rip, regs.rax, regs.rdi, regs.rsi, regs.rdx, regs.r10, regs.r8, regs.r9);
```

Comment:
A couple of times thru out this code and elsewhere we maintain the mapping between registers and syscalls args. Does it make sense to refactor with something like a `regs_to_syscall_args()` function?